dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/FirstOrderFilterLimitationDetection.mo

```diff
@@ -0,0 +1,68 @@
+  Modelica.Blocks.Logical.Greater greater1 annotation(
+    Placement(visible = true, transformation(origin = {30, -80}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Logical.LessEqual lessEqual annotation(
+    Placement(visible = true, transformation(origin = {140, 40}, extent = {{-10, 10}, {10, -10}}, rotation = 0)));
+  Modelica.Blocks.Logical.LessEqual lessEqual1 annotation(
+    Placement(visible = true, transformation(origin = {140, -40}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Logical.Or or1 annotation(
+    Placement(visible = true, transformation(origin = {170, 80}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Logical.Or or2 annotation(
+    Placement(visible = true, transformation(origin = {170, -80}, extent = {{-10, 10}, {10, -10}}, rotation = 0)));
+
+equation
+  connect(gain.y, greater1.u2) annotation(
+    Line(points = {{-97, 0}, {-80, 0}, {-80, -88}, {18, -88}}, color = {0, 0, 127}, pattern = LinePattern.Dash));
+  connect(limiter.y, greater1.u1) annotation(
+    Line(points = {{-41, 0}, {-20, 0}, {-20, -80}, {18, -80}}, color = {0, 0, 127}, pattern = LinePattern.Dash));
+  connect(gain.y, greater.u2) annotation(
+    Line(points = {{-98, 0}, {-80, 0}, {-80, 88}, {18, 88}}, color = {0, 0, 127}, pattern = LinePattern.Dash));
+  connect(limiter.y, greater.u1) annotation(
+    Line(points = {{-38, 0}, {-20, 0}, {-20, 80}, {18, 80}}, color = {0, 0, 127}, pattern = LinePattern.Dash));
+  connect(or2.y, fMin) annotation(
+    Line(points = {{181, -80}, {190, -80}, {190, -60}, {210, -60}}, color = {255, 0, 255}));
+  connect(or1.y, fMax) annotation(
+    Line(points = {{182, 80}, {190, 80}, {190, 60}, {210, 60}}, color = {255, 0, 255}));
+  connect(greater1.y, or2.u1) annotation(
+    Line(points = {{42, -80}, {158, -80}}, color = {255, 0, 255}));
+  connect(limiter1.y, lessEqual1.u1) annotation(
+    Line(points = {{122, 0}, {124, 0}, {124, -40}, {128, -40}}, color = {0, 0, 127}, pattern = LinePattern.Dash));
+  connect(limiter1.y, lessEqual.u1) annotation(
+    Line(points = {{122, 0}, {124, 0}, {124, 40}, {128, 40}}, color = {0, 0, 127}, pattern = LinePattern.Dash));
+  connect(integrator.y, lessEqual.u2) annotation(
+    Line(points = {{82, 0}, {90, 0}, {90, 48}, {128, 48}}, color = {0, 0, 127}, pattern = LinePattern.Dash));
+  connect(integrator.y, lessEqual1.u2) annotation(
+    Line(points = {{82, 0}, {90, 0}, {90, -48}, {128, -48}}, color = {0, 0, 127}, pattern = LinePattern.Dash));
+  connect(lessEqual1.y, or2.u2) annotation(
+    Line(points = {{152, -40}, {154, -40}, {154, -72}, {158, -72}}, color = {255, 0, 255}));
+  connect(lessEqual.y, or1.u2) annotation(
+    Line(points = {{152, 40}, {154, 40}, {154, 72}, {158, 72}}, color = {255, 0, 255}));
+  connect(greater.y, or1.u1) annotation(
+    Line(points = {{42, 80}, {158, 80}}, color = {255, 0, 255}));
+
```

Comment:
No preferred view ? Is it done by the extends ?

Reply:
It is not done by the extends, so I added it.